package/MDAnalysis/analysis/align.py

```diff
@@ -1002,23 +1006,66 @@ def sequence_alignment(mobile, reference, match_score=2, mismatch_penalty=-1,
         Tuple of top sequence matching output `('Sequence A', 'Sequence B', score,
         begin, end)`
 
+    Notes
+    -----
+    If you prefer to work directly with :mod:`Bio.Align` objects then you can
+    run your alignment with :class:`Bio.Alig.PairwiseAligner` as ::
+
+      import Bio.Align.PairwiseAligner
+
+      aligner = Bio.Align.PairwiseAligner(
+         mode="global",
+         match_score=match_score,
+         mismatch_score=mismatch_penalty,
+         open_gap_score=gap_penalty,
+         extend_gap_score=gapextension_penalty)
+      aln = aligner.align(reference.residues.sequence(format="Seq"),
+                          mobile.residues.sequence(format="Seq"))
+
+      # choose top alignment with highest score
+      topalignment = aln[0]
+
+    The ``topalignment`` is a :class:`Bio.Align.PairwiseAlignment` instance
+    that can be used in your bioinformatics workflows.
+
     See Also
     --------
-    BioPython documentation for `pairwise2`_. Alternatively, use
+    BioPython documentation for `PairwiseAligner`_. Alternatively, use
     :func:`fasta2select` with :program:`clustalw2` and the option
     ``is_aligned=False``.
 
-    .. _`pairwise2`: http://biopython.org/DIST/docs/api/Bio.pairwise2-module.html
+
+    .. _`PairwiseAligner`:
+       https://biopython.org/docs/latest/api/Bio.Align.html#Bio.Align.PairwiseAligner
+
 
     .. versionadded:: 0.10.0
 
+    .. versionchanged:: 2.4.0
+       Replace use of deprecated :func:`Bio.pairwise2.align.globalms` with
+       :class:`Bio.Align.PairwiseAligner`.
+
     """
-    aln = Bio.pairwise2.align.globalms(
-        reference.residues.sequence(format="string"),
-        mobile.residues.sequence(format="string"),
-        match_score, mismatch_penalty, gap_penalty, gapextension_penalty)
-    # choose top alignment
-    return aln[0]
+    aligner = Bio.Align.PairwiseAligner(
```

Comment:
The main argument for pairwisealigner is `scoring` do we want/need to pass that in somehow?

Reply:
If we are deprecating it then no, let's not add features. At this stage I'd just want to avoid the code suddenly breaking because pairwise2 is gone (and getting rid of DeprecationWarning).